src/amo/components/AddonReview.js

```diff
@@ -74,31 +84,38 @@ export class AddonReviewBase extends React.Component {
       );
     }
 
+    const titlePlaceholder = i18n.gettext('Give your review a title');
```

Comment:
Probably separate but I find it so weird we require titles in reviews; it'd be nice if failing receiving one we just extracted the first sentence or ignored it. I always hate being asked for titles in my reviews; makes me feel like I'm doing my High School English papers again.